core/src/physics.rs

```diff
@@ -56,6 +56,12 @@ pub struct KinematicBody {
     pub angular_velocity: f32,
     pub gravity: f32,
     pub bounciness: f32,
+    /// Sets a 1 frame override for the body friction. It will be re-set to `None` every frame so if
+    /// you wish to apply a continuous friction change, you must re-set it every frame.
+    ///
+    /// This is useful for things like slippery blocks or other things that want to modify a body's
+    /// friction while it is on the block.
+    pub frame_friction_override: Option<f32>,
```

Comment:
Awesome, thanks for the update!

I pushed a slight tweak that renamed the `friction` field and changed it to an `Option` to try and be more clear about how it's used.

I just wanted to make sure that looks good to you before merging.

Reply:
Looks good to me 👍